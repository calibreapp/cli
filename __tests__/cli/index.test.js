const execa = require('execa')

test('calibre --help', async () => {
  const { stdout } = await execa('calibre', ['--help'])
  expect(stdout).toMatchSnapshot()
})

test('calibre --version', async () => {
  const { stdout } = await execa('calibre', ['--version'])
  expect(stdout).toMatchSnapshot()
})
