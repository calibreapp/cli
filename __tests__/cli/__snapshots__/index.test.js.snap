// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calibre --help 1`] = `
"â™   calibre subcommand [options]

Commands:

  calibre site <command>   Use the site command to manage your Sites, Pages, Tes
                           t Profiles, Pull Request Reviews, Deploys, Snapshots,
                            and download monitoring data.
  calibre team <command>   Use the team command to manage Teams.
  calibre test <command>   Use the test command to run Single Page Tests: public
                            or private tests for a single Page tested from a sin
                           gle device. This command also supports viewing all Si
                           ngle Page Tests and retrieving their artifacts.
  calibre connection-list  List all available network connection speeds.
  calibre device-list      List all available test devices.
  calibre location-list    List all available test locations.
  calibre metric-list      List all available web performance metrics.
  calibre token <command>  Use the token command to store or revoke a Calibre AP
                           I token. You can also save your token with the CALIBR
                           E_API_TOKEN environment variable.
  calibre request          Use the request command to make a request to the Cali
                           bre GraphQL API.

Options:

  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]

Examples:

  calibre token set <token>  Store Calibre API token in ~/.config/configstore/ca
                             libre.json
  calibre site list          List the sites in your Calibre account
  calibre test create <url>  Create a single page test

For more information on Calibre, see https://calibreapp.com"
`;
